components/admin_console/permission_schemes_settings/permission_system_scheme_settings/permission_system_scheme_settings.jsx

```diff
@@ -440,6 +440,7 @@ export default class PermissionSystemSchemeSettings extends React.Component {
                         />
                     </BlockableLink>
                     <a
+                        data-testid='reset_permissions_to_default'
```

Comment:
this should be `camelCase`

Reply:
so looks like theres actually no standard for `testid` in the codebase. some places use `kebab-case` some places use `camelCase` but from what I can see theres nowhere that we use `snake_case` other than translation keys.

I would opt for `camelCase` mostly because our html IDs are also in `camelCase`

Reply:
Done thanks!